lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
+    <i className="fa fa-circle" style={{ color: props.color }}></i> { props.label }
+    { props.children }
+  </span>
+
+StatusIndicator.propTypes = {
+  color: PropTypes.string,
+  label: PropTypes.string,
```

Comment:
Something like this could work, basically check to see if the prop is either a string or a node 👍 

```
children: PropTypes.onOfType([
    PropTypes.string,
    PropTypes.node,
])
```